jcl/src/openj9.jvm/share/classes/com/ibm/jvm/DumpConfigurationUnavailableException.java

```diff
@@ -1,8 +1,10 @@
 /*[INCLUDE-IF Sidecar17]*/
 package com.ibm.jvm;
 
+import openj9.management.internal.DumpConfigurationUnavailableExceptionBase;
```

Comment:
Please update the include tag to `Sidecar18-SE` and move the `package` and `import` declarations after the copyright notice.

Reply:
Good plan.  Done.